front/app/components/FormBuilder/components/FormBuilderSettings/LinearAndRatingSettings/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { ICustomFieldInputType } from 'api/custom_fields/types';
+
+export const getNumberLabelForIndex = (
+  inputType: ICustomFieldInputType,
+  index: number
+) => {
+  if (inputType === 'sentiment_linear_scale') {
+    return index - 2; // We show a scale of -2 to 2 in the UI for sentiment scales.
+  }
+  return index + 1; // Show a scale of 1 to 11 in the UI for other field types.
+};
```

Comment:
Minor, I would define the scale offsets as it seems to be clearer for me but it could also be a preference so I will leave it to you

```suggestion
import { ICustomFieldInputType } from 'api/custom_fields/types';

const SCALE_OFFSETS: Partial<Record<ICustomFieldInputType, number>> = {
  sentiment_linear_scale: -2, // We show a scale of from -2 in the UI for sentiment scales.
};

export const getNumberLabelForIndex = (
  inputType: ICustomFieldInputType,
  index: number
) => {
  // Show a scale from 1 in the UI for other field types.
  return (SCALE_OFFSETS[inputType] ?? 1) + index;
};

```

Reply:
Ah - yes that's smart! Much clearer that way :+1: Thanks!